lib/configs/addon-interactions.ts

```diff
@@ -17,7 +17,7 @@ export = {
       },
     },
     {
-      files: ['main.@(js|cjs|mjs|ts)'],
+      files: ['.storybook/main.@(js|cjs|mjs|ts)'],
```

Comment:
These changes don't quite seem related to the PR, @zhyd1997 what prompted you to make these 3 changes?

Reply:
This is very likely correct. It's possible a previous PR didn't update these, and while they ran `yarn update-all` this got updated